test/prototype_transforms_kernel_infos.py

```diff
@@ -2232,6 +2232,22 @@ def sample_inputs_normalize_image_tensor():
         yield ArgsKwargs(image_loader, mean=mean, std=std)
 
 
+def reference_normalize_image_tensor(image, mean, std, inplace=False):
+    mean = torch.tensor(mean).view(-1, 1, 1)
+    std = torch.tensor(std).view(-1, 1, 1)
+
+    sub = torch.Tensor.sub_ if inplace else torch.Tensor.sub
+    return sub(image, mean).div_(std)
+
+
+def reference_inputs_normalize_image_tensor():
+    yield ArgsKwargs(
+        make_image_loader(size=(32, 32), color_space=datapoints.ColorSpace.RGB, extra_dims=[1]),
+        mean=[0.5, 0.5, 0.5],
+        std=[1.0, 1.0, 1.0],
+    )
```

Comment:
This is equivalent to what we had before, but of course we can also expand this.

Reply:
What do you mean by "what we had before"?

We have a bunch of tests for the current v1 `normalize()`. I assume we'll still keep these tests around even when we migrate?

Reply:
Yes, sorry. It is equivalent to the test you removed in #7113, i.e.

https://github.com/pytorch/vision/blob/c06d52b1c5f6aee36802661c3ebc6347b97cc59e/test/test_prototype_transforms_functional.py#L1188-L1197

The output type tests are covered by #7118.

Reply:
Before merging, should we make sure that all of what is convered in the v1 tests are also covered here? https://github.com/pytorch/vision/blob/01d138d81ae97b846ff63339eeee51f44a1725a2/test/test_transforms.py#L1948-L1999

e.g. the statistical test.

Or is it something we would do later - and in that case, how do we make sure to remember that?

Reply:
As discussed offline, all tests linked above except for the stats one were already covered and even more comprehensive so. I've added the missing stats test in 09b7fb30672e006c9e8758a9b6de8b8224415d70.